colorify.js

```diff
@@ -1,53 +1,63 @@
-BGCOLORS = [
-  "#e1f6f2", "#e5f2fa", "#f2eaf6", "#fdf7e0", "#fce8e6", "#f6f7f8"
-]
-
-function colorify() {
-  var previousColor;
-
-  $(".alert").each(function(_, e) {
-    var project = getProjectName(e);
-    var color = getColorForProject(project, previousColor);
-    previousColor = color;
-
-    e.style.backgroundColor = color;
-    e.style.paddingTop = "5px";
-    e.style.border = "5px solid " + color;
-  });
-}
-
-function getProjectName(e) {
-  name = $(e).find('.title > a:last').text().split('#')[0];
-  return name.split('@')[0];
-}
-
-function getColorForProject(project, previousColor) {
-  if (!(project in colors)) {
-    var color = BGCOLORS[Math.floor(Math.random() * BGCOLORS.length)];
-    if (color == previousColor) {
-      return getColorForProject(project, previousColor);
+(function (document, chrome) {
+
+  var colorify = {};
+
+  colorify.init = function () {
+    var self = this;
+
+    chrome.storage.sync.get('colors', function(result) {
+      if (Object.keys.call(null, result).length !== 0) {
+        self.colors = result.colors;
+      } else {
+        self.colors = {};
+      }
+    });
+
+    document.onreadystatechange = function () {
+      if (document.readyState == "complete") {
+        self.paint();
+        document.querySelector('.js-events-pagination').onclick = function () {
+          setTimeout(function () {
+            self.paint();
+          }, 1000);
```

Comment:
Is this going to refresh every second? Why you didn't like the `click` event on the "Show more" button?